main.py

```diff
@@ -13,9 +13,29 @@
                                              initial_distribution=v,
                                              num_stages=N, stopping_time=tau).create()
 
-tree.bulls_eye_plot(dot_size=5, radius=300)
+# tree.bulls_eye_plot(dot_size=5, radius=300)
+#
+# tree.set_data_at_node(4, {"a": 1})
+# print(sum(tree.probability_of_node(tree.nodes_at_stage(8))))
+#
+# print(tree)
 
-tree.set_data_at_node(4, {"a": 1})
-print(sum(tree.probability_of_node(tree.nodes_at_stage(8))))
+x0 = np.array([[1], [1]])
 
-print(tree)
+Aw1 = np.array([[1, 1], [1, 1]])
+Aw2 = np.array([[1, 1], [1, 1]])
+Aw3 = np.array([[1, 1], [1, 1]])
+As = [Aw1, Aw2, Aw3]
+
+Bw1 = np.array([[1, 1], [1, 1]])
+Bw2 = np.array([[1, 1], [1, 1]])
+Bw3 = np.array([[1, 1], [1, 1]])
+Bs = [Bw1, Bw2, Bw3]
+
+problem_config = r.core.RAOCPconfig(scenario_tree=tree)\
+    .with_As(As)\
+    .with_Bs(Bs)\
+    .with_all_cost_type("quadratic")\
```

Comment:
how do you specify the `Q` and `R` of the quadratic cost function? And is it Markovian?

Reply:
updated